modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,55 @@
+.Procedure
+
+. When using a JetBrains IDE requesting a registration, see xref:provisioning-jetbrains-activation-code-for-offline-use.adoc[].
+
+. Add the following component to the workspace devfile:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: "__<URL_to_meta.yaml>__" <1>
+----
++
+<1> __<URL_to_meta.yaml>__: HTTPS resource defining the editor metadata. It can be:
++
+* IntelliJ IDEA Community prebuilt image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml
+----
++
+* PyCharm Community prebuilt image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-pycharm/latest.meta.yaml
+----
++
+* Metadata for a custom editor image, see xref:building-intellij-editor-images.adoc[].
+
+. Remove the plugins or commands defined for the Theia editor from the workspace devfile.
+
+. Restart the workspace.
+
+. Read and accept the JetBrains Privacy Policy.
+
+. Make a choice in the IntelliJ data sharing preferences screen.
+
+. The IntelliJ IDE don't support checking out a repository using the devfile. Click the btn[New Project] to create a new project or the btn[Get from VCS] button to checkout a repository. 
+
+
+.Additional resources
+
+* xref:building-intellij-editor-images.adoc[]
+* xref:provisioning-jetbrains-activation-code-for-offline-use.adoc[]
+* link:https://github.com/che-incubator/jetbrains-editor-images/raw/meta/che-idea/latest.meta.yaml[Metadata for IntelliJ IDEA Community]
+* link:https://github.com/che-incubator/jetbrains-editor-images/raw/meta/che-pycharm/latest.meta.yaml[Metadata for PyCharm Community Edition]
+* link:https://quay.io/repository/che-incubator/che-idea?tab=tags[Container image for IntelliJ IDEA Community]
+* link:https://quay.io/repository/che-incubator/che-pycharm?tab=tags[Container image for PyCharm Community Edition]
```

Comment:
If you accept my suggestion above to include this link in the procedure text, then you can remove it from Additional resources here.